tests/unit/source_code_comment_test_files/test_file_1

```diff
@@ -71,3 +71,30 @@ void test_func(int num){ // line 70
     // wrong formatted suppress comment
     cout << "test func"  << endl;
 }
+
+/* codechecker_suppress [ my_checker_1 ] suppress comment */
+void test_func(int num){ // line 76
+    cout << "test func"  << endl;
+}
+
+/* codechecker_suppress [ my_checker_1 ]
+some longer
+comment */
+void test_func(int num){ // line 83
+    cout << "test func"  << endl;
+}
+
+/* codechecker_suppress [ my_checker_1 ]
```

Comment:
Could you please add a test case where there is a new line after the starting block comment:
```cpp
/*
  codechecker_suppress [ my_checker_1 ]
*/
void f() {}
```
This style is used a lot.